libs/beacon/src/beacon_service.cpp

```diff
@@ -234,7 +338,7 @@ BeaconService::State BeaconService::OnCollectSignaturesState()
                                       BeaconServiceProtocol::GET_SIGNATURE_SHARES, index);
 
   // Timer to wait maximally for network events
-  timer_to_proceed_.Restart(std::chrono::milliseconds{50});
+  timer_to_proceed_.Restart(std::chrono::milliseconds{200});
```

Comment:
This delay for network events was too small I think, causing failures in beta net